runner.go

```diff
@@ -273,6 +296,34 @@ func monitorRunStatus(c cliContext, runID int) error {
 	}
 }
 
+func buildRerunArgs(c cliContext, runID int) ([]string, error) {
+	maxReruns := c.Uint("max-reruns")
+	rerunAttempt := c.Uint("rerun-attempt")
+
+	cmd := []string{
+		"rainforest-cli",
+		"rerun", strconv.Itoa(runID),
+		"--max-reruns", fmt.Sprint(maxReruns),
+		"--rerun-attempt", fmt.Sprint(rerunAttempt + 1),
+		"--skip-update", // skip auto-updates for reruns
+	}
+
+	if token := c.GlobalString("token"); len(token) > 0 {
+		cmd = append(cmd, "--token", token)
+	}
+	if conflict := c.String("conflict"); len(conflict) > 0 {
+		cmd = append(cmd, "--conflict", conflict)
+	}
+	if background := c.Bool("background"); background {
+		cmd = append(cmd, "--background")
```

Comment:
This flag won't work with `--max-retries` for the same reasons as `--fail-fast` won't right? If you kick off a run with `--background`, the command will finish before the run does.

Reply:
Yes, good point, fixing.